buffer/src/main/java/io/netty/buffer/ByteBufUtil.java

```diff
@@ -66,6 +66,10 @@ protected CharBuffer initialValue() throws Exception {
 
     static final ByteBufAllocator DEFAULT_ALLOCATOR;
 
+    private static final String BOM_STR = "\uFEFF";
+    private static final byte[] BOM_UTF8 = BOM_STR.getBytes(CharsetUtil.UTF_8);
+    private static final byte[] BOM_US_ASCII = BOM_STR.getBytes(CharsetUtil.US_ASCII);
```

Comment:
Can you sanity check that this isn't just question marks like `??`

Reply:
I'm afraid I don't quite follow.  You mean check in the tests?

Reply:
Oh, I mean just print it out and see what the chars actually are.